src/levels.rs

```diff
@@ -112,12 +112,14 @@ pub const HOR_PRED: IntraPredMode = 2;
 pub const VERT_PRED: IntraPredMode = 1;
 pub const DC_PRED: IntraPredMode = 0;
 
-pub type InterIntraPredMode = c_uint;
 pub const N_INTER_INTRA_PRED_MODES: usize = 4;
```

Comment:
```suggestion
pub const N_INTER_INTRA_PRED_MODES: usize = InterIntraPredMode::COUNT;
```

Reply:
Actually, you can just delete this since it's not exported outside of the crate and replace its uses.